package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "@fractal-framework/fractal-contracts",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Why are we using this version? This implies the code is production ready. I would have started it at 0.1.0

Reply:
yeah if we can change it afer it's already been published, might make sense to bump it to 0.2.0